src/modules/auth/webauthn/WebAuthnScreen.js

```diff
@@ -292,4 +313,120 @@ function WebAuthnCard(props) {
 			}
 		</Card>
 	);
+
+
 }
+
+function TableRow (props) {
+	const { t, i18n } = useTranslation();
+	const [localEditMode, setLocalEditMode] = useState(false);
+	let obj = props.obj;
+
+	const cancelChanges = () => {
```

Comment:
@Aringoaway Pls add comments to every method you have created that it is clear for what purpose it is used